DataFormats/SiStripDetSetCompressedCluster/src/SiStripDetSetCompressedCluster.cc

```diff
@@ -0,0 +1,35 @@
+#include "DataFormats/SiStripDetSetCompressedCluster/interface/SiStripDetSetCompressedCluster.h"
+
+SiStripDetSetCompressedCluster::SiStripDetSetCompressedCluster() {
+  compressedAmplitudes_.clear();
+  firstStrip_.clear();
+  //error_x_.clear();
```

Comment:
Not needed.
This constructor can be removed and defined as ` = default;` in the class interface.